CSharp/Concentus.props

```diff
@@ -15,7 +15,7 @@
     <Authors>Logan Stromberg</Authors>
     <PackageProjectUrl>https://github.com/lostromb/concentus</PackageProjectUrl>
     <PackageIconUrl>http://durandal.dnsalias.net/imgur/concentus.png</PackageIconUrl>
-    <PackageLicenseUrl>https://opus-codec.org/license/</PackageLicenseUrl>
+    <PackageLicenseFile>LICENSE</PackageLicenseUrl>
```

Comment:
invalid XML - still using PackageLicenseUrl for the closing tag

Reply:
Whoops, sorry!